packages/commonjs/src/transform.js

```diff
@@ -288,7 +288,7 @@ export function transformCommonjs(
 
         if (flattened.keypath === 'module.exports' && node.right.type === 'ObjectExpression') {
           node.right.properties.forEach((prop) => {
-            if (prop.computed || prop.key.type !== 'Identifier') return;
+            if (prop.computed || !('key' in prop) || prop.key.type !== 'Identifier') return;
```

Comment:
nit:

I think this is more readable if we do something like `'key' in prop && `

Reply:
That wouldn't do what we want, if `key` is not in `prop` it would not satisfy this condition and would proceed to the next line. We want this condition to be true for `SpreadElement`.